src/access_coupler/mom_oasis3_interface.F90

```diff
@@ -678,7 +686,10 @@ subroutine from_coupler(step,Ocean_sfc,Ice_ocean_boundary, Time)
   case('lprec')
      Ice_ocean_boundary%lprec(iisc:iiec,jjsc:jjec) =  vwork(iisc:iiec,jjsc:jjec)
   case('salt_flx')
-     Ice_ocean_boundary%salt_flux(iisc:iiec,jjsc:jjec) =  vwork(iisc:iiec,jjsc:jjec)
+!     Ice_ocean_boundary%salt_flux(iisc:iiec,jjsc:jjec) =  vwork(iisc:iiec,jjsc:jjec)
+!dhb599 20170728: Nic Hanna's bug fix ==> change the i2o saltflux sign to "-"
+     Ice_ocean_boundary%salt_flux(iisc:iiec,jjsc:jjec) = - vwork(iisc:iiec,jjsc:jjec)
+!------
```

Comment:
Sign change in salt flux.

If suggested by Nic, then I'm not sure why it would not have been added, so maybe it was changed elsewherE?

Reply:
Yes, the negative sign is correct. We went over this pretty thoroughly.

Reply:
Yes, thanks @aidanheerdegen sorry if that wasn't clear Russ

Reply:
It is included, as of july last year. https://github.com/mom-ocean/MOM5/commit/f2a120dd67663c7c35e983ffad8e6c0f58af216f#diff-a8a52883e9d375e61a5188fee48c7556

Reply:
Yeah, I see it... are we diffing the right thing here??

Reply:
I don't understand, this doesn't even seem to be the right diff. The current code is here:

https://github.com/mom-ocean/MOM5/blob/master/src/access_coupler/mom_oasis3_interface.F90#L689

And contains a long comment. I am confused.